src/translation/translation_fr_FR.ts

```diff
@@ -1617,7 +1617,7 @@ Nous avons coupé votre canal et activé &quot;Me silencer&quot;. Veuillez d&apo
         <location filename="../clientsettingsdlg.cpp" line="200"/>
         <location filename="../clientsettingsdlg.cpp" line="222"/>
         <source>Some sound card drivers do not allow the buffer delay to be changed from within %1. In this case the buffer delay setting is disabled and has to be changed using the sound card driver. Use the appropriate tool for the interface in use to adjust this buffer size. For example, if using ASIO, use the &quot;ASIO Device Settings&quot; button to open the driver settings panel or if using JACK, use a tool such as QjackCtl to adjust the buffer size. Other interfaces, such as Pipewire, would require their appropriate tool being used. Please refer to the interface manual.</source>
-        <translation type="unfinished"></translation>
+        <translation>Certains pilotes de carte son ne permettent pas de modifier le délai de la mémoire tampon depuis %1. Dans ce cas, le réglage de délai de tampon est désactivé et doit être modifié à l&apos;aide du pilote de la carte son. Utilisez l&apos;outil approprié pour l&apos;interface utilisée afin de régler la taille de la mémoire tampon. Par exemple, si vous utilisez ASIO, utilisez le bouton &quot;Configuration du pilote&quot; pour ouvrir le panneau des paramètres du pilote ou si vous utilisez JACK, utilisez un outil tel que QjackCtl pour ajuster la taille du tampon. D&apos;autres interfaces, telles que Pipewire, nécessitent l&apos;utilisation de leur propre outil. Veuillez vous référer au manuel de l&apos;interface.</translation>
```

Comment:
For `filename="../clientsettingsdlg.cpp" line="403"`, `"ASIO Device Settings"` is translated as `"Paramètres du périphérique ASIO"` whereas here it's `"Configuration du pilote"`, which is "Driver Setup" from `filename="../clientsettingsdlgbase.ui" line="394"`.  As it's a button label, it needs to be the right one.

Reply:
It's ok for me to translate from the english button. For now, it is labeled `Driver Setup`.
![driver-setup](https://user-images.githubusercontent.com/12748768/177208859-abda5c9c-b37e-4c3f-82ea-56586907f159.png)
I think there is a problem with this text instead of with the button.

Reply:
![asio-button](https://user-images.githubusercontent.com/12748768/177209880-ea2f6c6a-281d-4d81-8bf7-afa36821492a.png)

Reply:
@pljones what about that ?

Reply:
Note that it's the _specific_ text for the case of ASIO that's being refered to here:
![image](https://user-images.githubusercontent.com/1549463/177382344-ecab2e5f-6a00-4e7d-96f0-cadcd276b536.png)

There's no button when Jack is in use, for example:
![image](https://user-images.githubusercontent.com/1549463/177383316-a831da0e-9528-402d-b1ce-f5228ffb03fd.png)